scripts/jenkins/ardana/ansible/roles/input_model_generator/defaults/main.yml

```diff
@@ -21,6 +21,12 @@ qe_ardana_bm_info_dir: "/tmp/ardana-bm-info"
 template_file_name: "{{ scenario[item ~ '_template'] }}.yml"
 input_model_dir: "/tmp/ardana-im/{{ scenario_name }}"
 
+ardana_ntp_servers:
```

Comment:
On a second thought, maybe this isn't really the best place to keep this variable. It isn't really an input model generator setting, but more like a global Ardana setting, which will be used by several roles (e.g. the heat generator, the input model generator). I would suggest moving this in the `cloud-common` role and including it here. When the legacy `init.yml` playbook will be merged with some of the QE playbooks, we can also include the same file there.

Reply:
As we discussed, I have moved it to group_vars/all as `ntp_servers` is a variable that will be shared between different playbooks/roles.